.github/workflows/ci-build.yml

```
@@ -6,6 +6,10 @@ on:
   pull_request:
     branches: [ main ]
 
+env:
+  CVE_CACHE_DIR: ~/.m2/repository/org/owasp/dependency-check-data/9.0/**
```

(minor) Any reason not to:

```suggestion
  CVE_CACHE_DIR: ~/.m2/repository/org/owasp/dependency-check-data/**
```